kuma/javascript/src/header/dropdown.jsx

```diff
@@ -69,7 +104,9 @@ export default function Dropdown(props: DropdownProps) {
             </button>
             <ul
                 id={props.ariaOwns}
-                className={`dropdown-menu-items${props.right ? ' right' : ''}`}
+                className={`dropdown-menu-items${props.right ? ' right' : ''} ${
```

Comment:
Extra space here `''} ${`

Reply:
I agree, the space is actually added inside the `true` portion but, I am not a fan :) I would rather just do:

```suggestion
className={`dropdown-menu-items ${props.right ? 'right' : ''} ${
```

as I did for the next `class` :+1:

Reply:
Oh doh! I misread what you were referring to. Either way, updated it to the latter so it is consistent with other places in the codebase.